.appveyor.yml

```diff
@@ -49,22 +50,21 @@ install:
   - set PATH=%CONDA_INSTALL_LOCN%;%CONDA_INSTALL_LOCN%\scripts;%PATH%;
   - set PYTHONUNBUFFERED=1
   - conda config --set always_yes true
-  - conda update --all
   - conda config --set show_channel_urls yes
   - conda config --prepend channels conda-forge
-  # this is now the downloaded conda...
-  - activate
-  - conda info -a
 
   # For building, use a new environment which only includes the requirements for mpl
   # if conda-forge gets a new pyqt, it might be nice to install it as well to have more backends
   # https://github.com/conda-forge/conda-forge.github.io/issues/157#issuecomment-223536381
   - conda create -q -n test-environment python=%PYTHON_VERSION%
-    freetype=2.6 tk=8.5
-    pip setuptools numpy sphinx tornado
+    pip setuptools wheel tk
   - activate test-environment
   - echo %PYTHON_VERSION% %TARGET_ARCH%
-  - pip install -r requirements/testing/travis_all.txt -r requirements/testing/travis36.txt
+  # Install dependencies from PyPI.
+  - python -mpip install --upgrade -r requirements/testing/travis_all.txt %EXTRAREQS% %PINNEDVERS%
+  # Install optional dependencies from PyPI.
+  # Sphinx is needed to run sphinxext tests
+  - python -mpip install --upgrade sphinx
```

Comment:
should this just go into EXTRAREQS?

Reply:
I do not know, probably then GUIs also should go into EXTRAREQS